lib/compiler/src/v3_core.erl

```diff
@@ -171,12 +172,16 @@
 
 -record(imodule, {name = [],
 		  exports = ordsets:new(),
-                  nifs = sets:new([{version, 2}]),
+                  nifs = false ::
+                    'false' | sets:set(), % a set if the attribute is
+                                          % present in the module.
```

Comment:
I suggest adding `is` for clarity:

```suggestion
                    'none' | sets:set(), %is a set if the attribute is
                                          %present in the module.
```

Also, in the compiler we prefer to use `none` instead of `false` in this situation.

Reply:
Fixed, although it prevents the cute use of field's value as a truth value at line 202 :)